sudospawner/mediator.py

```diff
@@ -50,12 +49,19 @@ def kill(pid, signal):
     app_log.debug("Sending signal %i to %i", signal, pid)
     try:
         os.kill(pid, signal)
-    except OSError as e:
-        if e.errno == errno.ESRCH:
-            # not running
-            alive = False
-        else:
-            raise
+    except ProcessLookupError:
+        # not running
+        alive = False
+    except PermissionError:
```

Comment:
I'm on the fence, too. I'll add a verbose catch-all log. If it really is failing, it will still look like the server failing to start, so people ought to be looking at the logs.